packages/cli/test/bin/bin.test.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-unused-vars */
+// packages/cli/test/bin.parseargs.test.js
+import { spawn } from "node:child_process";
+import { strict as assert } from "node:assert";
+import { fileURLToPath } from "node:url";
+import { dirname, join } from "node:path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const binPath = join(__dirname, "../../src/bin.js");
+
+function runCLI(args = [], timeoutMs = 10000) {
```

Comment:
I'm looking into adding a feature to the gallinago package so you can use the same cli runner here that the rest of the test suite uses.

https://github.com/ProjectEvergreen/greenwood/blob/e72c8061c6b4a14dbac18e236ab9bc80d34d3395/packages/cli/test/cases/build.config.layouts-directory/build.config.layouts-directory.spec.js#L51